Source/WebKit/WebProcess/Network/WebTransportReceiveStreamSource.cpp

```diff
@@ -44,21 +44,38 @@ WebTransportReceiveStreamSource::~WebTransportReceiveStreamSource()
     ASSERT(RunLoop::isMain());
 }
 
-void WebTransportReceiveStreamSource::receiveBytes(std::span<const uint8_t> bytes, bool)
+void WebTransportReceiveStreamSource::receiveBytes(std::span<const uint8_t> bytes, bool withFin, std::optional<WebCore::Exception>&& exception)
 {
     ASSERT(RunLoop::isMain());
-    if (m_isCancelled)
+    if (m_isCancelled || m_isClosed)
+        return;
+    if (exception) {
+        controller().error(*exception);
+        clean();
```

Comment:
I'm not very familiar with these calls.  Do you know what happens when clean is not called?